src/wgpu.zig

```diff
@@ -1347,8 +1347,8 @@ pub const Buffer = *opaque {
     }
     extern fn wgpuBufferGetConstMappedRange(buffer: Buffer, offset: usize, size: usize) ?*const anyopaque;
 
-    // `offset` has to be a multiple of 8 (otherwise `null` will be returned).
-    // `@sizeOf(T) * len` has to be a multiple of 4 (otherwise `null` will be returned).
+    // `offset` - in bytes, has to be a multiple of 8 (otherwise `null` will be returned).
+    // `len` - length of slice to return, in elements of type T, `@sizeOf(T) * len` has to be a multiple of 4 (otherwise `null` will be returned).
```

Comment:
Thanks!
We should probably use doc-comments `///` here so this gets displayed in users editors.